config/default.yml

```
@@ -3422,6 +3422,11 @@ Style/OneLineConditional:
   VersionAdded: '0.9'
   VersionChanged: '0.38'
 
+Style/OpenSSLDigestConstant:
```

I think warnings for deprecated methods belong to the `Lint` department.

```suggestion
Lint/OpenSSLDigestConstant:
```